cmd/network/evpn-utils.go

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (c) 2022-2023 Intel Corporation, or its subsidiaries.
+// Copyright (c) 2022-2023 Dell Inc, or its subsidiaries.
+// Copyright (c) 2024 Ericsson AB.
+
+// Package network implements the network related CLI commands
+package network
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/PraserX/ipconv"
+	pb "github.com/opiproject/opi-api/network/evpn-gw/v1alpha1/gen/go"
+	pc "github.com/opiproject/opi-api/network/opinetcommon/v1alpha1/gen/go"
+)
+
+// PrintComponents prints the components with their details
+func PrintComponents(comp []*pb.Component) string {
+	var status string
+	for i := 0; i < len(comp); i++ {
+		str := fmt.Sprintf(" %+v\n", comp[i])
+		status += str
+	}
+	return status
+}
+
+// PrintGWIPs prints the gw ips
+func PrintGWIPs(comp []*pc.IPPrefix) string {
+	var status string
+	for i := 0; i < len(comp); i++ {
+		str := fmt.Sprintf("%+v/%v ", ipconv.IntToIPv4(comp[i].GetAddr().GetV4Addr()), comp[i].GetLen())
+		status += str
+	}
+	return status
+}
+
+// PrintLB prints the logical bridge fields in human readable format
+func PrintLB(lb *pb.LogicalBridge) {
+	Vteip := fmt.Sprintf("%+v/%v", ipconv.IntToIPv4(lb.GetSpec().GetVtepIpPrefix().GetAddr().GetV4Addr()), lb.GetSpec().GetVtepIpPrefix().GetLen())
+	log.Println("name:", lb.GetName())
+	log.Println("status:", pb.LBOperStatus_name[int32(lb.GetStatus().GetOperStatus())])
```

Comment:
to print enum as string you can just use `.String()` method
```go
log.Println("status:", lb.GetStatus().GetOperStatus().String())
```

or you can try to call even not to call it explicitly
```go
log.Println("status:", lb.GetStatus().GetOperStatus())
```